docs/configuration/operator-args.rst

```diff
@@ -92,7 +92,7 @@ dbt-related
 - ``models``: Specifies which nodes to include.
 - ``no_version_check``: If set, skip ensuring ``dbt``'s version matches the one specified in the ``dbt_project.yml``.
 - ``quiet``: run ``dbt`` in silent mode, only displaying its error logs.
-- ``vars``: (Deprecated since Cosmos 1.3 use ``ProjectConfig.dbt_vars`` instead) Supply variables to the project. This argument overrides variables defined in the ``dbt_project.yml``.
+- ``vars``: Supply dbt variables to run the task using dbt project. This argument overrides variables defined in the ``dbt_project.yml`` and any values set in ``ProjectConfig.dbt_vars``.
```

Comment:
This is nice that we mention that this will override projectconfig.dbt_vars

Do you think it would also make sense to add somewhere in this document that operator_args are meant only for execution time and they are not meant for DAG parsing/rendering?

Reply:
Improved in 08e4a294fd0270b1ed8232e602539ab3dc77cfd8